diff --git a/chrome/android/java/src/org/chromium/chrome/browser/searchwidget/SearchWidgetProvider.java b/chrome/android/java/src/org/chromium/chrome$
index 92792cdab427..adbba9848072 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/searchwidget/SearchWidgetProvider.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/searchwidget/SearchWidgetProvider.java
@@ -108,9 +108,9 @@ public class SearchWidgetProvider extends AppWidgetProvider {
     }

     static final String ACTION_START_TEXT_QUERY =
-            "org.chromium.chrome.browser.searchwidget.START_TEXT_QUERY";
+           "com.presearch.search.TextSearchActivity";
     static final String ACTION_START_VOICE_QUERY =
-            "org.chromium.chrome.browser.searchwidget.START_VOICE_QUERY";
+           "com.presearch.search.VoiceSearchActivity"; //Action added by Mamy
     static final String ACTION_UPDATE_ALL_WIDGETS =
             "org.chromium.chrome.browser.searchwidget.UPDATE_ALL_WIDGETS";
