diff --git a/components/embedder_support/android/java/src/org/chromium/components/embedder_support/util/UrlConstants.java b/components/embedder_support/android/java/src/org/chromium/components/embedder_support/util/UrlConstants.java 
index 18ef55238234..e7582962714c 100644
--- a/components/embedder_support/android/java/src/org/chromium/components/embedder_support/util/UrlConstants.java
+++ b/components/embedder_support/android/java/src/org/chromium/components/embedder_support/util/UrlConstants.java
@@ -45,7 +45,7 @@ public class UrlConstants {
     public static final String CHROME_BLANK_URL = "chrome://blank/";
 
     public static final String NTP_HOST = "newtab";
-    public static final String NTP_URL = "chrome-native://newtab/";
+    public static final String NTP_URL = "https://engine.presearch.org"; //Changed by Mamy
     public static final String NTP_NON_NATIVE_URL = "chrome://newtab/";

     public static final String BOOKMARKS_HOST = "bookmarks";

